p2p/relay.go

```diff
@@ -92,29 +92,35 @@ func NewRelayReserver(tcpNode host.Host, relay *MutablePeer) lifecycle.HookFunc
 			// When the reservation expires, the server needs to re-reserve.
 			// When the connection expires (stream reset error), then client needs to reconnect.
 
+			refreshDelay := time.Until(resv.Expiration.Add(-time.Minute * 2))
```

Comment:
```suggestion
			refreshDelay := time.Until(resv.Expiration.Add(-2 * time.Minute))
```
which one should be preferred?